platform/features/timeline/src/controllers/swimlane/TimelineSwimlaneDropHandler.js

```diff
@@ -169,7 +190,7 @@ define(
                     return inEditMode() &&
                         target &&
                         !pathContains(target, id) &&
-                        allowsCompose(target, domainObject);
+                        !!chooseAction(target.domainObject, domainObject);
```

Comment:
When will chooseAction return undefined? Seems as though it's only if actionCapability does not exist?